gocd/pipelines/relay-pop.yaml

```diff
@@ -23,7 +23,28 @@ pipelines:
         stage: promote-to-pops
 
     stages:
-      # checks aren't needed because it's implied by pipeline-deploy-relay-completed.
+      - checks:
+          approval:
+            type: manual
+          fetch_materials: true
+
+          jobs:
+            checks:
+              environment_variables:
+                # Required for checkruns.
+                GITHUB_TOKEN: "{{SECRET:[devinfra-github][token]}}"
+              timeout: 1800
+              elastic_profile_id: relay
```

Comment:
Should this be the same profile as processing relays?

Reply:
It's now `relay-pop` to align with the other pop jobs